### ISSUE TITLE
LibreSSL: Update ObsoleteFiles.inc

### ISSUE THREAD

**Issue by lattera** [HardenedBSD/hardenedBSD-playground#2]
For the LibreSSL branch, we should add the old OpenSSL files to ObsoleteFiles.inc.

**Comment by Sp1l** [comment]
Looking into this that would require a switch to default to WITH_LIBRESSL as well?

**Comment by lattera** [comment]
We have the LIBRESSL knob added to the __DEFAULT_NO_OPTIONS array in src.opts.mk as it stands right now. I don't think we'd need to switch it to __DEFAULT_YES_OPTIONS, though that might be a good idea for the libressl branch, given the point of the libressl branch is for... libressl (tada!).

But you bring up a good point. Right now, we're not actually keying off the LIBRESSL knob correctly right now. We should fully hook up to the src.opts.mk framework and key off of MK_LIBRESSL instead of WITH_LIBRESSL. I'll add another ticket. Only complete this ticket after the new one is completed as this one will depend on that one.

**Comment by lattera** [comment]
This issue depends on #3.

**Comment by Sp1l** [comment]
libcrypto man-pages to be added 
[OpenSSL-ObsoleteFiles.inc.txt](https://github.com/HardenedBSD/hardenedBSD-playground/files/253424/OpenSSL-ObsoleteFiles.inc.txt)

```
OLD_LIBS+=lib/libcrypto.so.7
OLD_LIBS+=usr/lib/libssl.so.7
OLD_LIBS+=usr/lib32/libcrypto.so.7
OLD_LIBS+=usr/lib32/libssl.so.7
OLD_LIBS+=lib/libcrypto.so.8
OLD_LIBS+=usr/lib/libssl.so.8
OLD_LIBS+=usr/lib32/libcrypto.so.8
OLD_LIBS+=usr/lib32/libssl.so.8
```
